renderdoc/core/image_viewer.cpp

```diff
@@ -667,7 +674,8 @@ void ImageViewer::RefreshFile()
   if(dds)
   {
     FileIO::fseek64(f, 0, SEEK_SET);
-    read_data = load_dds_from_file(f);
+    StreamReader reader(f);
```

Comment:
Same thing here with `FILE` ownership changing hands to the `StreamReader`.

Reply:
Same as above.